src/classes/USER_UserService_TEST.cls

```diff
@@ -0,0 +1,223 @@
+        System.assertEquals(USER_UserService.ERROR_NOTIFICATION_RECIPIENT_ALL_SYS_ADMINS, errorSettingsSysAdmin.Error_Notifications_To__c);    
+    }
+
+
+    /*********************************************************************************************************
+    * @description Tests setting the NPSP Settings Error Notification Recipients to 'All Sys Admins'.
+    * @return void
+    **********************************************************************************************************/
+    private static testMethod void testSetErrorNotificationToAllSysAdmins() {
+        List<User> newUsers = createUsers();
+        Set<Id> sysAdminIds = new Map<Id, User>(newUsers).keySet();
+
+        UTIL_CustomSettingsFacade.getErrorSettingsForTests(new Error_Settings__c(Error_Notifications_On__c = true, 
+            Error_Notifications_To__c = newUsers[1].Id, Store_Errors_On__c = true));
+        Error_Settings__c errorSettings = UTIL_CustomSettingsFacade.getErrorSettings();
+        Database.insert(errorSettings, true);
+
+        Test.startTest();
+        USER_UserService.setErrorNotificationToAllSysAdminsForInActiveSysAdmin(sysAdminIds);
+        Test.stopTest();
+
+        Error_Settings__c errorSettingsSysAdmin = UTIL_CustomSettingsFacade.getErrorSettings();
+        System.assertEquals(USER_UserService.ERROR_NOTIFICATION_RECIPIENT_ALL_SYS_ADMINS, errorSettingsSysAdmin.Error_Notifications_To__c);
+    }
+
+
+    /*********************************************************************************************************
+    * @description Tests if System Administrator is the owner of Scheduled Jobs and calls the methods that
+    *              log an error and send an email.
+    * @return void
+    **********************************************************************************************************/
+    private static testMethod void testHandleScheduledJobsSubmittedByInActiveSysAdmin() {
+        List<User> newUsers = createUsers();
+        Set<Id> sysAdminIds = new Map<Id, User>(newUsers).keySet();
+
+        List<CronTrigger> jobs = [SELECT Id FROM CronTrigger];
+        for (CronTrigger eachJob : jobs) {
+            System.abortJob(eachJob.Id);
+        }
+
+        System.runAs(newUsers[0]) {
+            UTIL_MasterSchedulableHelper.setScheduledJobs();
+        }
+
+        List<CronTrigger> scheduledJobs = [SELECT Id, OwnerId, CronExpression, CronJobDetail.Name, CreatedBy.Name, CreatedBy.IsActive 
+                                            FROM CronTrigger
+                                            WHERE CronJobDetail.JobType = :USER_UserService.JOB_TYPE_SCHEDULED_APEX];
+
+        Test.startTest();
+        USER_UserService.handleScheduledJobsSubmittedByInActiveSysAdmin(sysAdminIds);
+        Test.stopTest();
+
+        if (!scheduledJobs.isEmpty()) {
+            System.assertEquals(USER_UserService.NUMBER_OF_SCHEDULED_JOBS, scheduledJobs.size());
```

Comment:
as mentioned earlier, no real value in this assert, and it introduces a dependency on keeping this constant in sync with the actual scheduled job table, that we periodically add to.

Reply:
Sounds good, I'll remove this assert.